Aptoide/aptoide-client-v8
# [ASV-1987] added new spend appc icon and color to the desired places;

**What does this PR do?**

This PR adds the new look for appc spend on home, search and appview

**Database changed?**

No

**How should this be manually tested?**

Check all the places and make sure everythin is alright (if you remember anywhere else that's not described in the ticket you can check there as well)

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1987](https://aptoide.atlassian.net/browse/ASV-1987)

**Code Review Checklist**
